src/_shared/utils/applyCurlyQuotes.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Helper to replace opening and closing curly single and double quotes
+ *
+ * @param text
+ * @returns {string}
+ */
+
+export const applyCurlyQuotes = (text: string): string => {
+  if (!text) return '';
```

Comment:
nit pick: We try not to omit the `{}` for a if block even if it is a single line 😅